src/timely-util/src/operator.rs

```diff
@@ -143,6 +144,11 @@ where
     /// Take a Timely stream and convert it to a Differential stream, where each diff is "1"
     /// and each time is the current Timely timestamp.
     fn pass_through<R: Data>(&self, name: &str, unit: R) -> Stream<G, (D1, G::Timestamp, R)>;
+
+    /// Wraps the stream with a passthrough operator that will be shut down when the provided
+    /// token cannot be upgraded anymore. After shutdown all data flowing into the operator will be
+    /// dropped.
+    fn with_token(&self, token: Weak<()>) -> Stream<G, D1>;
```

Comment:
I think this comment is no longer accurate? The operator does not shut down per se, but rather consumes all its input without producing output.

Reply:
Fixed!